testnet/libp2p_config.yaml

```diff
@@ -21,7 +23,7 @@ network:
     log_level: "fatal"
     node_key: "0000000000000000000000000000000000000000000000000000000000000001"
     discV5BootstrapNodes: []
-    initial_peers: []
+    initial_peers: ["/dns/bootstrap/tcp/3000"]
     relayTopics: []
     # Mixclient configuration to communicate with mixnodes.
     # The libp2p network backend always requires this mixclient configuration
```

Comment:
One nomos-node must be run with `mixnet_client.mode: !SenderReceiver mix-node-2:7778`, so that the nomos-node can receive messages from the `mix-node-2` (which is in the last hop of mixnet) and can broadcast it to other nomos-nodes via gossipsub. 
https://github.com/logos-co/nomos-node/tree/master/mixnet#recommended-architecture

I think it might be simpler to run the bootstrap nomos-node as a `SenderReceiver`, so that any other subsequent nomos-nodes can subscribe messages broadcasted from the bootstrap node.

Reply:
Updated the bootstrap node to have this in it's config, thanks for pointing that out. Would it make sense to have all nodes in SenderReceiver mode?

Reply:
Also, thank you for making this change: https://github.com/logos-co/nomos-node/pull/467/commits/8f442005a5febeafef698de5d3af248b716ed73c#diff-157e6ba3a88af9687c0e336e3c95ff7fbacef1e8720d828ed4a0d041b5494b5a. I had to test it before..